appserver/payara-appserver-modules/environment-warning/src/main/java/fish/payara/appserver/environment/warning/config/EnvironmentWarningConfiguration.java

```diff
@@ -59,7 +59,7 @@
     String getMessage();
     public void setMessage(String message);
     
-    @Attribute(required = false, defaultValue = "#325764")
+    @Attribute(required = false, defaultValue = "#ff0000")
```

Comment:
Very minor point here, but could we have a "nice" red colour? E.g. the Bootstrap red shade is #D9534F